docs/release_notes.adoc

```diff
@@ -13,40 +15,73 @@ include::include.adoc[]
+* `@TempDir` `spock.tempDir.keep` has been replaced by `spock.tempdir.cleanup`. See <<extensions.adoc#_temp_dir_cleanup,TempDir Cleanup>> spockPull:1525[]
 
 === Misc
-* Add support for <<extensions.adoc#extension-store,keeping state in extensions>>
 * Add support for parameter injection of `@TempDir`
-* Add `@Snapshot` extension for <<extensions.adoc#snapshot-testing,snapshot testing>>
-* Add <<extensions.adoc#spock-interceptors,feature-scoped interceptors>> spockPull:1844[]
+* Add cleanup switch for `@TempDir` spockPull:1525[]
+* Add support for creation of global Groovy mocks for abstract classes spockPull:1754[]
+* Add optional reason to `@ResourceLock` spockPull:1890[]
+* Add optional reason to `@Execution` spockPull:1576[]
+* Add `onTimeout` support for `PollingConditions` spockPull:1853[]
+* Add `SpecInfo#getAll...Methods()` methods spockPull:1770[]
+* Add array support to collection conditions spockPull:1734[]
+* Add support for logging thread dumps when <<extensions.adoc#timeout-extension,@Timeout interrupts>> are ignored spockPull:1658[]
+* Add implied features to force execution of features even when removed by a build tool spockPull:1598[]
+* Improve StackTraceFilter by ignoring `java.lang.invoke.*` spockPull:1892[]
 * Improve `@TempDir` field injection, now it happens before field initialization, so it can be used by other field initializers.
+* Improve Spock-Compiler does not use wrapper types anymore spockPull:1765[]
+* Improve reduce lock contention of the `byte-buddy` mock maker, when multiple mocks are created concurrently spockPull:1778[]
+* Improve `@Use` on feature and fixture method for parallel execution spockPull:1691[]
+* Improve IDE support
+** by adding a closure signature hint that derives closure argument type from variable type spockPull:1785[]
+** by adding missing closure hints for Spy(T, Closure) spockPull:1786[]
+** by adding GDSL/DSLD for ConditionalExtension's annotations spockPull:1808[]
+** by making `spock.gdsl` fail-safe and cover some more cases spockPull:1783[]
+* Improve RunContexts behavior in multithreaded executions spockPull:1758[], spockPull:1846[]
+* Improve generic handling by replacing `gentyref` code with https://github.com/leangen/geantyref[geantyref] library spockPull:1743[]
+* Improve handling of passing `null` to `thrown()` to be consistent with mock object creation spockPull:1799[]
+* Improve `old(...)` calls by validating their argument count spockPull:1810[]
+* Improve robustness of the AST transformation against missing classes in the compile classpath spockPull:1704[]
+* Improve handling of `shared.` conditions in `@IgnoreIf`/`@Requires`, the condition is now checked before creating data providers spockPull:1711[]
+** This is now a required dependency used by spock: `io.leangen.geantyref:geantyref:1.3.14`
```

Comment:
The line 61 `This is now a required` should be moved to line 57 after the `Improve generic handling by replacing`

Reply:
good catch